pkg/metrics/volumesnapshotschedule.go

```diff
@@ -0,0 +1,107 @@
+package metrics
+
+import (
+	"fmt"
+
+	snapv1 "github.com/kubernetes-incubator/external-storage/snapshot/pkg/apis/crd/v1"
+	stork_api "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	snapshotcontrollers "github.com/libopenstorage/stork/pkg/snapshot/controllers"
+	"github.com/prometheus/client_golang/prometheus"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	volumeSnapshotScheduleStatusCounter = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "stork_volume_snapshotschedule_status",
+		Help: "Status of volume snapshot schedules",
+	}, []string{metricName, metricNamespace, metricPolicy})
+	volumeSnapshotStatusCounter = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "stork_volume_snapshot_status",
+		Help: "Status of volume snapshots",
+	}, []string{metricName, metricNamespace, metricSchedule})
+)
+
+var (
+	// volumeSnapshotStatus map of volume snapshot status to enum
+	volumeSnapshotStatus = map[snapv1.VolumeSnapshotConditionType]float64{
+		snapv1.VolumeSnapshotConditionReady:   0,
+		snapv1.VolumeSnapshotConditionError:   1,
+		snapv1.VolumeSnapshotConditionPending: 2,
+	}
+)
+
+func watchVolumeSnapshotScheduleCR(object runtime.Object) error {
+	volumeSnapshotSchedule, ok := object.(*stork_api.VolumeSnapshotSchedule)
+	if !ok {
+		err := fmt.Errorf("invalid object type on volume snapshot schedule watch: %v", object)
+		return err
+	}
+	labels := make(prometheus.Labels)
+	labels[metricName] = volumeSnapshotSchedule.Name
+	labels[metricNamespace] = volumeSnapshotSchedule.Namespace
+	labels[metricPolicy] = volumeSnapshotSchedule.Spec.SchedulePolicyName
+	suspend := volumeSnapshotSchedule.Spec.Suspend
+
+	if volumeSnapshotSchedule.DeletionTimestamp != nil {
+		volumeSnapshotScheduleStatusCounter.Delete(labels)
+		return nil
+	}
+	if *suspend {
```

Comment:
should we switch this behavior 
- if suspended then set to 1
- else set to 0

This matches with snapshot ready - 0

Reply:
sounds good, had it switched.